index.html

```diff
@@ -45,6 +45,174 @@ <h1>
       <img src="images/Normal Button/Secondary/Disabled.png" alt="">
     </section>
   </section>
+  <section class="r_works">
+    <section class="title">
+      <p>My Recent Works</p>
```

Comment:
- [OPTIONAL] As this was not pointed out by the previous reviewer, I am going to make this optional, But I'll advise you to implement them before merging this branch. :smile: :+1: 
- Following the best semantics practice, This is a title and it should have a `hx tag eg. h2, h3` :+1:

Reply:
Ok done